apps/wallet-mobile/src/features/Swap/common/LiquidityPool/LiquidityPool.tsx

```diff
@@ -25,20 +28,26 @@ export const LiquidityPool = ({
   )
 }
 
-const styles = StyleSheet.create({
-  liquidityPoolLink: {
-    alignItems: 'center',
-    justifyContent: 'center',
-  },
-  liquidityPoolText: {
-    color: '#4B6DDE',
-    fontFamily: 'Rubik',
-    fontSize: 16,
-    fontWeight: '500',
-    lineHeight: 22,
-  },
-  liquidityPool: {
-    flexDirection: 'row',
-    alignItems: 'center',
-  },
-})
+const useStyles = () => {
+  const {theme} = useTheme()
+  const {color} = theme
+  const styles = StyleSheet.create({
+    liquidityPoolLink: {
+      alignItems: 'center',
+      justifyContent: 'center',
+    },
+    liquidityPoolText: {
+      color: color.primary[500],
```

Comment:
let's later check all `Text`s components, it should be matching figma, therefore any `Text` must have a typography attached to it, not a blocker to merge it.